modules/nf-core/hamronization/abricate/tests/main.nf.test

```diff
@@ -0,0 +1,58 @@
+nextflow_process {
+
+    name "Test Process HAMRONIZATION_ABRICATE"
+    script "../main.nf"
+    process "HAMRONIZATION_ABRICATE"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "hamronization"
+    tag "hamronization/abricate"
+
+    test("hamronization/abricate - bacteroides_fragilis - genome_abricate_tsv") {
+
+        when {
+            process {
+                """
+                input[0] = [ [ id:"test" ], file(params.test_data['bacteroides_fragilis']['hamronization']['genome_abricate_tsv'], checkIfExists: true) ]
+                input[1] = 'tsv'
+                input[2] = '1.0.1'
+                input[3] = '2021-Mar-27'
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out).match() }
+            )
+        }
+
+    }
+
+    test("hamronization/abricate - stub") {
+
+        options "-stub"
+
+        when {
+            process {
+                """
+                input[0] = [ [ id:"test" ],file("dummy.tsv") ]
+                input[1] = ''
+                input[2] = ''
+                input[3] = ''
```

Comment:
Couldn't you use the same inputs as the main test? What you have works, I'm just curious why you chose to have them remain different!

Reply:
I wondered that too.. but then had same thought, 'true not necessarily needed, nm' :laughing:

Reply:
I didn't have a strong opinion about those, but @ahvigil has (see below comment), so I went for same inputs for stub and normal tests now ;)